aips/aip-119.md

```diff
@@ -0,0 +1,230 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Scheduled Transactions
+author: Manu Dhundi (https://github.com/manudhundi) & Zekun Li (https://github.com/zekun000)
+discussions-to (*optional):
+Status: Draft
+last-call-end-date (*optional):
+type: Core, Framework
+created: 04/15/2025
+updated (*optional):
+requires (*optional): <AIP number(s)>
+---
+
+# AIP-119 - Scheduled Transactions
+  
+(Please give a temporary file name to your AIP when first drafting it, such as `aip-x.md`. The AIP manager will assign a number to it after reviewing.)
+
+(Please remove the questions in the "quote box". Provide complete context to the questions being asked in the content that you provide.)
+
+## Summary
+A scheduled transaction is a user-submitted transaction that specifies another transaction to be executed at a future time. The system executes it in a suitable future block.  
```

Comment:
nit: I would make this statement generic, and then say we are only targeting time based scheduling in the AIP, e.g.:

```
A scheduled transaction is a user-submitted transaction that specifies another transaction to be
executed by a specific trigger, e.g., a time-based or event-based trigger. The system executes the
transaction in a suitable future block.  

This AIP focuses on time-based transaction triggers (other triggers will be left for future AIPs). 
In the AIP, we propose framework changes to support scheduling...
```